build.gradle

```diff
@@ -81,13 +81,14 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
     implementation 'com.github.j4velin.colorpicker:colorpicker:1.20.6'
-    implementation 'com.github.j4velin.EazeGraph:EazeGraph:1.0.3'
+    implementation 'com.github.blackfizz:eazegraph:1.2.5l@aar'
+    implementation 'com.nineoldandroids:library:2.4.0'
```

Comment:
Could you explain why you'd need a new library? Also, I had some additions in my fork of the EazeGraph lib, is there any specific reason to change to this other version of the lib?